channels/my.m3u

```diff
@@ -29,6 +29,8 @@ http://210.210.155.35/uq2663/h/h07/01.m3u8
 http://stream2.ninmedia.tv/nabawitv/nabawitv_360p/playlist.m3u8
 #EXTINF:-1 tvg-id="NTV7" tvg-name="NTV7" tvg-language="" tvg-logo="https://freeview.github.io/iptv/logos/ntv7.png" group-title="",NTV7
 https://raw.githubusercontent.com/exodiver/IPTV/master/M3U8/Token/NTV7.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://yt3.ggpht.com/a/AATXAJzd2RPKopBOqInaP7TGFkpOzRgbwaApn81UBA" group-title="Entertainment",Pertiwi TV
+https://mn-nl.mncdn.com/pertiwitv/pertiwitv/playlist.m3u8
```

Comment:
The link is not working at the moment. Tested in streamtest.in, Elmedia Video Player and VLC player.
<img width="937" alt="Captura de pantalla 2020-05-28 a las 12 25 52" src="https://user-images.githubusercontent.com/7253922/83124269-a767c100-a0de-11ea-803e-b98ac51f5df1.png">
<img width="608" alt="Captura de pantalla 2020-05-28 a las 12 26 01" src="https://user-images.githubusercontent.com/7253922/83124272-a9318480-a0de-11ea-905a-ebdd5e642d67.png">
<img width="363" alt="Captura de pantalla 2020-05-28 a las 12 26 23" src="https://user-images.githubusercontent.com/7253922/83124275-aafb4800-a0de-11ea-911d-959161df8d81.png">

Reply:
That is strange, perhaps a temporary downtime? I've just checked it now in VLC, ffmpeg, and videoJS. Seems to be working ok.

Reply:
You're probably right. The channel's working fine right now.